snapcraft.yaml

```diff
@@ -10,6 +10,11 @@ confinement: strict
 icon: icon.png
 
 parts:
+  gadget-yaml:
+    plugin: nil
+    override-build: |
+      # snapcraft reads gadget.yaml from project dir
+      install -m 644 gadget-$SNAPCRAFT_TARGET_ARCH.yaml $SNAPCRAFT_PROJECT_DIR
```

Comment:
doesn't this need to install it as "gadget.yaml"?

Reply:
Thanks! I've just pushed a commit to fix this; I don't know how to fully test the snap part of this branch, so I might have missed other issues there